tests/integration/test_config.py

```diff
@@ -43,17 +46,53 @@ def get_field_value(*names: str, configs: Iterable[ObjectConfig | HostGroupConfi
     )
 
 
+async def get_service_with_config(cluster: Cluster) -> Service:
+    return await cluster.services.get(name__eq="complex_config")
+
+
+async def get_object_config(obj: Service | ConfigHostGroup, httpx_client: AsyncClient) -> dict:
+    url = "/".join(map(str, (*obj.get_own_path(), "configs", "")))
+    response = await httpx_client.get(url=url, params={"ordering": "-id", "limit": 2})
+    assert response.status_code == 200
+
+    for result in response.json()["results"]:
+        if result["isCurrent"]:
+            cfg_id = result["id"]
+            break
+    else:
+        raise RuntimeError("Can't find current config")
+
+    response = await httpx_client.get(url=f"{url}/{cfg_id}/")
```

Comment:
imo url is tailed with / already, so no need in duplicating it

Reply:
fixed